core/logging_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import (
+	"testing"
+
+	"github.com/op/go-logging"
+	"github.com/spf13/viper"
+)
+
+func TestLoggingLevelDefault(t *testing.T) {
+	viper.Reset()
+
+	LoggingInit("")
+
+	assertDefaultLoggingLevel(t, DefaultLoggingLevel())
+}
+
+func TestLoggingLevelOtherThanDefault(t *testing.T) {
+	viper.Reset()
+	viper.Set("logging_level", "warning")
+
+	LoggingInit("")
+
+	assertDefaultLoggingLevel(t, logging.WARNING)
+}
+
+func TestLoggingLevelForSpecificModule(t *testing.T) {
```

Comment:
A 'module' in this case is just an arbitrary string that gets passed to the logger which can be used when formatting the log statement. Doing it with different values would just exercise the same logic with the different value.

I can add some different values though if you would like.